documentation/docs/configuration.md

```diff
@@ -315,6 +318,38 @@ if [`get5_print_damage`](#get5_print_damage) is disabled.<br>
 ####`get5_knife_cfg`
 :   Config file executed for the knife round, relative to `csgo/cfg`.<br>**`Default: "get5/knife.cfg"`**
 
+####`get5_autoload_config`
+:  A [match configuration](../match_schema/#schema) file, relative to the `csgo` directory, to autoload when a player
+joins the server if no match is loaded. Set to empty string to disable. **`Default: ""`**
+
+## Demos
+
+####`get5_demo_upload_url`
+:   If defined, it is the URL at which a server resides to accept connections to upload demos. Requires
+[SteamWorks](../installation/#steamworks) extension. If no protocol is provided, the plugin will prepend
+`http://` to this value. **`Default: ""`**
+
+####`get5_demo_upload_header_key`
+:   If defined, it is the authorization key that is appended to the HTTP request. Requires
+[SteamWorks](../installation/#steamworks) extension. **`Default: Authorization`**
+
+####`get5_demo_upload_header_value`
+:   If defined, it is the authorization value that is appended to the HTTP request. Requires
+[SteamWorks](../installation/#steamworks) extension. This is where your API *key* would
+be set. **`Default: ""`**
+
+####`get5_demo_upload_delete_after_upload`
+:   Whether or not to delete the demo from the game server after successfully uploading
+it to a web server. Requires [`get5_demo_upload_url`](./#get5_demo_upload_url) to be set.
+**`Default: 0`**
+
+####`get5_demo_path`
+:   The folder of saved [demo files](../gotv), relative to the `csgo` directory. You **can** use
+the [`{MATCHID}`](#tag-matchid) and [`{DATE}`](#tag-date) variable, i.e. `demoes/{DATE}/{MATCHID}/`.
+Much like [`get5_backup_path`](./#get5_backup_path), the path must **not** start with a slash, and
+must **end with a slash**.
+**`Default: ""`**
+
```

Comment:
Move `get5_demo_name_format` docs down here, so we group all demo-related stuff. It's above under `Formats & Paths` currently.